samples/rag-cosmosdb/python/host.json

```diff
@@ -9,8 +9,19 @@
     "openai": {
       "searchProvider": {
         "type": "cosmosDBSearch",
+        "connectionString": "",
         "vectorSearchDimensions": 1536,
-        "numLists": 1
+        "numLists": 1,
+        "kind": "VectorIVF",
+        "similarity": "Cosine",
+        "numberOfConnections": 16,
+        "efConstruction": 64,
+        "efSearch": 40,
+        "databaseName": "functionsDatabase",
+        "collectionName": "functionsCollection",
+        "indexName": "functionsIndex",
```

Comment:
comment in discussion for collection name, database name, index name and connection string at other place.